Pythoncore/coding.py

```diff
@@ -0,0 +1,36 @@
+# find common substring in list of string
+def find_long_subs(l1):
+    first=l1[0]
+    temp=[]
```

Comment:
find a optimized solution. It has 3 inner loops which makes this solution efficient.